string status
---
Pose station
string command